doc/dev/mgmt/tests.md

```diff
@@ -15,178 +14,136 @@ IMPORTANT NOTE: All the commands in this page assumes you have loaded the [dev_s
 
 # Writing new tests
 
-SDK tests are based on the `scenario_tests` subpackage of the
-[`azure-devtools`](https://pypi.python.org/pypi/azure-devtools) package.
-`scenario_tests` is a general, mostly abstract framework
-providing several features useful for the SDK tests, for example:
+SDK tests are based on the `scenario_tests` subpackage of the [`azure-devtools`](https://pypi.python.org/pypi/azure-devtools) package. `scenario_tests` is a general, mostly abstract framework providing several features useful for the SDK tests, for example:
 
-*   HTTP interaction recording and playback using [vcrpy](https://pypi.python.org/pypi/vcrpy)
-*   Creation and cleanup of helper resources, such as resource groups,
-    for tests that need them in order to test other services
-*   Processors for modifying requests and responses when writing or reading recordings
-    (for example, to avoid recording credential information)
-*   Patches for overriding functions and methods that don't work well with tests
-    (such as long-running operations)
+* HTTP interaction recording and playback using [vcrpy](https://pypi.python.org/pypi/vcrpy)
+* Creation and cleanup of helper resources, such as resource groups, for tests that need them in order to test other services
+* Processors for modifying requests and responses when writing or reading recordings (for example, to avoid recording credential information)
+* Patches for overriding functions and methods that don't work well with tests (such as long-running operations)
 
-Code in the [`azure-sdk-tools/devtools_testutils`](https://github.com/Azure/azure-sdk-for-python/tree/master/tools/azure-sdk-tools/devtools_testutils) directory
-provides concrete implementations of the features provided in `scenario_tests`
-that are oriented around use in SDK testing
-and that you can use directly in your unit tests.
+Code in the [`azure-sdk-tools/devtools_testutils`](https://github.com/Azure/azure-sdk-for-python/tree/master/tools/azure-sdk-tools/devtools_testutils) directory provides concrete implementations of the features provided in `scenario_tests` that are oriented around use in SDK testing and that you can use directly in your unit tests.
 
 ## Test structure
 
-New tests should be located alongside the packages containing the code they test.
-For example, the tests for `azure-mgmt-media` are in `azure-mgmt-media/tests`.
-Each test folder also has a `recordings` subfolder containing one .yaml recording file
-for each test that has HTTP interactions to record.
+New tests should be located alongside the packages containing the code they test. For example, the tests for `azure-mgmt-media` are in `azure-mgmt-media/tests`. Each test folder also has a `recordings` subfolder containing one `.yaml` recording file for each test that has the HTTP interactions for the single test (with the authentication secrets scrubbed from the recordings).
```

Comment:
We should remove mention of "recordings", or clarify that this folder is auto-created. Right now it gives the impression you have to do something to make this folder exists